explorer/client/src/utils/static/mock_data.json

```diff
@@ -443,6 +443,58 @@
                     }
                 }
             }
+        },
+        {
+            "hI1m16cQkwErCDqbPyuV2ZtX4DZEel2lQTU1b1VJq0c=": {
+                "transaction": {
+                    "data": {
+                        "kind": {
+                            "Single": {
+                                "Transfer": {
+                                    "recipient": "ca27601ec5d915dd40d42e36c395d4a156b24026",
```

Comment:
Could this address be replaced by `0xa5e6dbcf33730ace6ec8b400ff4788c1f150ff7e` or any other address we have data for in `mock_data.json`? This would enable testing the ability of the user to click on an address referred to in transaction results in order to go to that Address Results page.